feat(dedibox-console): Adding tunnel Dedibox server termination ext-add-dedibox-console-howto (#4111)
scaleway/docs-content: dedibox-console/account/how-to/terminate-dedibox-server.mdx

> ---

Reviewer: I think this page should be in the dedibox/dedicated-servers/ section (rather than dedibox-console/acount).

This new page also needs to be added to the menu (json file, here: https://github.com/scaleway/docs-content/blob/main/menu/navigation.json) 

If you like, I can close this PR and I'm happy to redo it myself in the right section and add it to the menu in that branch? Or feel free to do it all yourself if you prefer:-)

Reviewer 2: ping @Jonathan-alter  :)

meta:
  title: How to terminate a Dedibox server under Online console

>   description: Learn how to terminate a Dedibox server under online console.

Reviewer: ```suggestion
  description:Learn how to terminate a Dedibox server in the Online console.
```

content:
  h2: How to terminate a Dedibox server

>   paragraph: This guide explains how to terminate a Online Dedibox server.

Reviewer: ```suggestion
  paragraph: This guide explains how to terminate a Dedibox server in the Online console.
```

> tags: dedibox terminate server online console

Reviewer: ```suggestion
tags: dedibox terminate server online console delete
```

dates:

>   validation: 2024-12-12

Reviewer: ```suggestion
  posted: 2024-12-12
  validation: 2024-12-12
```

categories:
  - dedibox-console
---

<Macro id="requirements" />
- A Dedibox account logged into the [console](https://console.online.net)
- An active Dedibox server

## How to terminate your Dedibox server

> 1. Click your username, next to _logged in as_, in the top right corner of the console. From the drop-down menu, select **Services**. The service page displays.

Reviewer: ```suggestion
1. Click your username, next to _logged in as_, in the top right corner of the [Online console](https://console.online.net/). From the drop-down menu, select **Services**. The **Services** page displays.
```

> 2. Click the **detail** of the **Dedibox server** you want to terminate. The **Server information** page will display.

Reviewer: ```suggestion
2. Click **Detail** for the Dedibox server you want to terminate. The **Server information** page displays.
```

> 3. Click **Terminate**. A pop-up will display.

Reviewer: ```suggestion
3. Click **Terminate**. A pop-up displays.
```

> 4. Click **Confirm Termination**. The Server and all options associated will display

Reviewer: ```suggestion
4. Click **Confirm Termination**. The server and all associated options displays.
```

> 5. Type comment in the **Reason for termination** field and choose a summary into the summary

Reviewer: ```suggestion
5. Type a comment in the **Reason for termination** field, and enter a summary.
```

6. Click **TERMINATE THIS SERVICE AND OPTION ASSOCIATED** to confirm the termination

<Message type="important">

> There are specific conditions for terminating a Dedibox server on the timing of your request.

Reviewer: ```suggestion
Your Dedibox server will be terminated in accordance with the timing of your termination request:
```
